install/github/mvn/workflows/publication.yml

```diff
@@ -6,198 +6,117 @@ on:
         uses: actions/setup-java@v1
         with:
-          java-version: 11
-      #~ MAVEN CACHE
+          java-version: ${{ env.JDK_VERSION }}
+
       - name: "Cache Maven packages"
+        id: cache-maven
         uses: actions/cache@v2
         with:
           path: ~/.m2
           key: ${{ runner.os }}-m2
           restore-keys: ${{ runner.os }}-m2
-      #~ BUILD CACHE
-      - name: "Cache Build Dir"
-        id: build-cache
-        uses: actions/cache@v2
-        with:
-          path: "${{ env.LOCAL_CLONE_DIR }}/target/"
-          key: ${{ runner.os }}-build-${{ env.BUILD_CACHE_KEY }}
-      #~ SCAN
-      - name: "Scan docker"
-        env:
-          ENTANDO_OPT_SNYK_ORG: "${{ secrets.ENTANDO_OPT_SNYK_ORG }}"
-          SNYK_TOKEN: "${{ secrets.SNYK_TOKEN }}"
-          ENTANDO_OPT_DOCKER_BUILDS: "${{ secrets.ENTANDO_OPT_DOCKER_BUILDS }}"
-          ENTANDO_OPT_DOCKER_ORG: "${{ secrets.ENTANDO_OPT_DOCKER_ORG }}"
-        run: |
-          ~/ppl-run docker scan --id "SCAN-DOCKER" --lcd "$LOCAL_CLONE_DIR"
 
-  post-dep-tests:
-    needs: [ 'publication' ]
-    if: needs.publication.outputs.POST_DEP_TESTS == 'true'
-    runs-on: ubuntu-latest
-    steps:
-      #~ CHECKOUT
-      - name: "CHECKOUT"
-        id: CHECKOUT
+      - name: Build with Maven
+        run: ${{ env.BUILD_COMMANDS }}
+
+      - name: Setting up variables
         run: |
-          ${{ secrets.ENTANDO_OPT_PPL_INSTALL_CMD }}
-          ~/ppl-run \
-            .. checkout-branch --id "CHECKOUT FOR PUBLICATION" \
-                  --lcd "$LOCAL_CLONE_DIR" \
-                  --token "$ENTANDO_BOT_TOKEN" \
-            .. pr-preflight-checks --only flags --lcd "$LOCAL_CLONE_DIR" \
-          ;
-          ~/ppl-run generic GENERATE-BUILD-CACHE-KEY "BUILD_CACHE_KEY" --lcd "$LOCAL_CLONE_DIR" >> $GITHUB_ENV
-      #~ JDK
-      - name: "Set up JDK 11"
-        uses: actions/setup-java@v1
+          echo "MAVEN_GROUP_ID=$(mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.groupId -q -DforceStdout)" >> $GITHUB_ENV
+          echo "MAVEN_ARTIFACT_ID=$(mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.artifactId -q -DforceStdout)" >> $GITHUB_ENV
+          echo "MAVEN_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.version -q -DforceStdout)" >> $GITHUB_ENV
+          echo "Debug 1"
+          REF_VERSION=${GITHUB_REF_NAME:1}
+          echo "Debug 2"
+          echo "$REF_VERSION"
+          REF_VERSION=${REF_VERSION/+*}
+          echo "Debug 3"
+          echo "$REF_VERSION"
+          echo "REF_VERSION=$REF_VERSION" >> $GITHUB_ENV
+          echo "Debug 4"
+
+      - name: Publish to Nexus
+        uses: sonatype-nexus-community/nexus-repo-github-action@1.0.0
         with:
-          java-version: 11
-      #~ MAVEN CACHE
-      - name: "Cache Maven packages"
-        uses: actions/cache@v2
+          serverUrl: https://nexus-jx.apps.serv.run/
```

Comment:
Can you substitute this static value with this ENV_VAR: NEXUS_INSTANCE_URI

This way is simpler to change the value globally if we have to